src/chainstate/stacks/db/blocks.rs

```diff
@@ -3001,15 +3035,86 @@ impl StacksChainState {
         }
     }
 
+    /// Get the highest processed block.  Break ties on lexigraphical ordering of the block hash
+    /// (i.e. arbitrarily).  The staging block will be returned, but no block data will be filled
+    /// in.
+    pub fn get_stacks_chain_tip(&self) -> Result<Option<StagingBlock>, Error> {
```

Comment:
I think this function should use util::db::query_row()

Reply:
Fixed in 0cbbfb1